plugins/ibm_qradar/help.md

```diff
@@ -65,8 +65,9 @@ Example input:
 ```
 {
```

Comment:
Can you please add an example with subfield. That will be very useful.

Reply:
Added for possible actions. This action does not have sub fields